pkg/operator/hive/hive_controller.go

```diff
@@ -56,6 +56,9 @@ const (
 	hiveConfigName = "hive"
 
 	hiveOperatorDeploymentName = "hive-operator"
+
+	// Daemonset name
+	daemonsetName = "hiveadmission"
```

Comment:
legacyDaemonsetName might help someone avoid confusion in future. Also the godoc comment is probably invalid.